### ISSUE TITLE
Allow user to display numbers as percentages

### ISSUE THREAD

**Issue by seancolsen** [centerofci/mathesar#1340]
## Motivation

Users should be able to store numbers as percentages.

## Current behavior

- The Mathesar Number type has a display option "Show as Percentage". The option is saved but it is not yet utilized for any behavior.

## Desired behavior

- The exact implementation is still TBD.

## Status

- Blocked by #1331 where we may decide to change plans from our initial implementation strategy.

**Comment by seancolsen** [comment]
Closing in favor of #1396 which is more specific
